### PR TITLE
Add folder reference table

### LINKED ISSUE #60: Add ability to run tests with smart folder name
It would be nice to be able to specify a smart folder name to run a group of tests.

### PR DESCRIPTION
**Pull request by epaulet**
fixes #60 by adding an easy reference for folder ids

A lot of the changes are copy-pasta because I renamed a class to be more generic

### CODE REVIEW

**Comment by epaulet** [comment]
On the terminal:

screen shot 2016-04-18 at 11 10 43 am

**Comment by epaulet** [comment]
@curtis-rainforestqa can i have a review please?

File: spec/resources_spec.rb [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+        # second line of text
+        expect(RainforestCli.logger).to receive(:info) do |message|
+          expect(message).to include('/settings/sites')
+        end
+
+        subject.sites
+      end
+    end
+
+    context 'with folders in account' do
```
Comment (curtis-rainforestqa) [inline spec/resources_spec.rb]: should `folders` be `sites` here?

**Comment by curtis-rainforestqa** [comment]
looks good! just one potential typo i commented on, other than that, lgtm, 👍

**Pull request merged**